app/controller_services/shopping_list_items_controller/destroy_service.rb

```diff
@@ -33,10 +33,6 @@ def master_list
       @master_list ||= shopping_list_item.list.master_list
     end
 
-    def master_list_item
```

Comment:
This method wasn't being used anywhere but did have the same name as a variable used elsewhere in the file.